README.md

```diff
@@ -27,9 +27,54 @@ described in the [integration guide](http://developer.gini.net/gini-sdk-ios/docs
 is provided at the instance's `injector` property) to create the manager instances and to manage the dependencies
 between the managers and makes those manager instances available as properties.
 
-## Requirements
+## Public Key pinning
+The Gini iOS SDK allows you to enable public key pinning with the Gini API through the [TrustKit](https://github.com/datatheorem/TrustKit/) library. In case you want to implement it, you just need to pass a dictionary with all the parameters required into one of the `GINISDKBuilder` initializers, as follows:
+
+**Swift**
+```swift
+let trustKitConfig = [
```

Comment:
In the Android documentation I show a working configuration with the current *.gini.net public key and an invalid key hash I created with `$ echo "0" | openssl dgst -sha256 -binary | openssl enc -base64`. Although it will get out-dated it could be more useful now for our clients to have a valid example.

Reply:
First I had it with "gini.net", but I decided to make it more generic. You are right, It could be better to make it more specific in this case.